dll/win32/browseui/internettoolbar.cpp

```diff
@@ -642,6 +643,21 @@ void CInternetToolbar::AddDockItem(IUnknown *newItem, int bandID, int flags)
     newSite->Initialize(newItem, this, fMainReBar, bandID, flags);
 }
 
+HRESULT CInternetToolbar::EnumBands(UINT Index, int *pBandId, IUnknown **ppUnkBand)
+{
+    REBARBANDINFOW rbbi;
+    rbbi.cbSize = sizeof(rbbi);
+    rbbi.fMask = RBBIM_ID | RBBIM_LPARAM;
+    rbbi.cch = 0;
+    if (!::SendMessageW(fMainReBar, RB_GETBANDINFOW, Index, (LPARAM)&rbbi))
+        return HRESULT_FROM_WIN32(ERROR_NO_MORE_ITEMS);
+    *pBandId = rbbi.wID;
+    if (!rbbi.lParam)
+        return E_UNEXPECTED;
+    *ppUnkBand = ((CDockSite*)(rbbi.lParam))->GetContainedBand(); // Not ref. counted
```

Comment:
Is this "not ref-counted" behaviour confirmed to happen on Windows?

Reply:
This function is internal. It's used to internally enumerate the bands **and** it helps to implement the IBandSite methods.